src/UI/Layout/SplitSelectable.elm

```diff
@@ -20,30 +19,6 @@ type alias Config object msg =
     }
 
 
-type alias MobileConfig msg =
-    { action : Maybe (Action.WithIcon msg)
-    , title : ( String, Maybe String )
-    , unselectMsg : msg
-    }
-
-
-mobile : RenderConfig -> MobileConfig msg -> Config object msg -> Nav.Content msg
```

Comment:
:warning:  **WARNING** :warning:  **WARNING** :warning:  **WARNING** :warning:  **WARNING** :warning: 

We're using this in WMS, as well as `MobileConfig`, the thing is that we forgot to add it to `exposed-modules` in `elm.json`